docs/modeling/example-fitting-model-sets.rst

```diff
@@ -1,37 +1,120 @@
 Fitting Model Sets
 ==================
 
-Fitting a polynomial model to multiple data sets simultaneously::
+Astropy model sets let you fit the same (linear) model to lots of independent
+data sets. It does the looping for you, and can be significantly faster than
+doing the looping yourself. But getting the data into the right shape can be 
+a bit tricky. 
 
-      >>> from astropy.modeling import models, fitting
-      >>> import numpy as np
-      >>> p1 = models.Polynomial1D(3)
-      >>> p1.c0 = 1
-      >>> p1.c1 = 2
-      >>> print(p1)
-      Model: Polynomial1D
-      Inputs: ('x',)
-      Outputs: ('y',)
-      Model set size: 1
-      Degree: 3
-      Parameters:
-           c0  c1  c2  c3
-          --- --- --- ---
-          1.0 2.0 0.0 0.0
-      >>> x = np.arange(10)
-      >>> y = p1(x)
-      >>> yy = np.array([y, y])
-      >>> p2 = models.Polynomial1D(3, n_models=2)
-      >>> pfit = fitting.LinearLSQFitter()
-      >>> new_model = pfit(p2, x, yy)
-      >>> print(new_model)  # doctest: +SKIP
-      Model: Polynomial1D
-      Inputs: 1
-      Outputs: 1
-      Model set size: 2
-      Degree: 3
-      Parameters:
-           c0  c1         c2                 c3
-          --- --- ------------------ -----------------
-          1.0 2.0 -5.86673908219e-16 3.61636197841e-17
-          1.0 2.0 -5.86673908219e-16 3.61636197841e-17
+In the example below, we create a 3D data cube where the first dimension is a ramp, 
+for example as from non-destructive readouts of an IR detector. So each pixel has a 
+depth along a time axis, and flux that results a total number of counts that is 
+increasing with time. We will be fitting a 1D polynomial vs. time to estimate the 
+flux in counts/second (the slope of the fit).
+
+   .. plot::
+       :include-source:
+
+       import numpy as np
+       import matplotlib.pyplot as plt
+       from scipy import stats
+       from astropy.modeling import models, fitting
+       
+       # Set up the shape of the image and create the time axis
+       DEPTH,WIDTH,HEIGHT=10,101,102 # Time is along the DEPTH axis
+       t = np.arange(DEPTH,dtype=np.float64)*10. # e.g. readouts every 10 seconds
+       
+       # Make up a flux in each pixel 
+       fluxes = np.arange(1.*WIDTH*HEIGHT).reshape(HEIGHT,WIDTH)
```

Comment:
```suggestion
       fluxes = np.arange(WIDTH * HEIGHT, dtype=np.float).reshape(HEIGHT, WIDTH)
```

Reply:
Added plot labels, but didn't extend the plots to the future as that seemed like making the example more complicated than it needed to be. 

Ran flake8 and addressed all the suggestions.